pybess/protobuf_to_dict.py

```diff
@@ -154,8 +154,14 @@ def _dict_to_protobuf(pb, value, type_callable_map, strict):
 
     for field, input_value, pb_value in fields:
         if field.label == FieldDescriptor.LABEL_REPEATED:
-            if field.message_type and field.message_type.has_options and field.message_type.GetOptions().map_entry:
-                pb_value.update(input_value)
+            if (field.message_type and field.message_type.has_options and
```

Comment:
Ah, yes, you need parens, `\`-at-end-of-line, or squash-it-all-into-one-line. Personally I prefer the parentheses 😄 but I suppose it is "sixes" as some say.